tests/test_array_utils.py

```diff
@@ -328,26 +332,893 @@ def test_transpose_color_in_array_with_batch_size_should_work() -> None:
+                        [8, 44, 80],
+                        [9, 45, 81],
+                        [10, 46, 82],
+                        [11, 47, 83],
+                        [12, 48, 84],
+                    ],
+                    [
+                        [13, 49, 85],
+                        [14, 50, 86],
+                        [15, 51, 87],
+                        [16, 52, 88],
+                        [17, 53, 89],
+                        [18, 54, 90],
+                    ],
+                    [
+                        [19, 55, 91],
+                        [20, 56, 92],
+                        [21, 57, 93],
+                        [22, 58, 94],
+                        [23, 59, 95],
+                        [24, 60, 96],
+                    ],
+                    [
+                        [25, 61, 97],
+                        [26, 62, 98],
+                        [27, 63, 99],
+                        [28, 64, 100],
+                        [29, 65, 101],
+                        [30, 66, 102],
+                    ],
+                    [
+                        [31, 67, 103],
+                        [32, 68, 104],
+                        [33, 69, 105],
+                        [34, 70, 106],
+                        [35, 71, 107],
+                        [36, 72, 108],
+                    ],
+                ]
+            ]
+        )
+    )
+
+    assert array.shape == (1, 3, 6, 6)
+    assert expected_array.shape == (1, 6, 6, 3)
+
+    array = transpose_color_last_in_array_pt(array=array)
+
+    assert array.shape == expected_array.shape
+    torch.testing.assert_allclose(array, expected_array)
+
+
+def test_transpose_color_first_in_array_pt_should_add_additional_dimensions() -> None:
+    array = torch.tensor(
+        np.array(
+            [
+                [
+                    [1],
+                    [2],
+                    [3],
+                    [4],
+                    [5],
+                    [6],
+                ],
+                [
+                    [7],
+                    [8],
+                    [9],
+                    [10],
+                    [11],
+                    [12],
+                ],
+                [
+                    [13],
+                    [14],
+                    [15],
+                    [16],
+                    [17],
+                    [18],
+                ],
+                [
+                    [19],
+                    [20],
+                    [21],
+                    [22],
+                    [23],
+                    [24],
+                ],
+                [
+                    [25],
+                    [26],
+                    [27],
+                    [28],
+                    [29],
+                    [30],
+                ],
+                [
+                    [31],
+                    [32],
+                    [33],
+                    [34],
+                    [35],
+                    [36],
```

Comment:
Is it automatic formatting? It's hardly readable to me.

Reply:
Yes, it is automatic.

Reply:
Let it be then.